tools/hidl/hidlcomp.cpp

```diff
@@ -3153,6 +3153,9 @@ void EspMessageInfo::write_esp_ipp()
         outf("\n\tC%s(IEspContext* ctx, const char *serviceName, IProperties *params, MapStrToBuf *attachments);", name_);
     }   
 
+    if (espm_type_==espm_request)
+        outs("\n\tIEspClientRpcSettings &rpc(){return *static_cast<IEspClientRpcSettings*>(this);}\n\n");
```

Comment:
rpc() is just providing access to the RPC (remote procedure call) part of the object it's a common term for these types of objects.  The rpc() reference will contain well named methods that affect the RPC communication, etc.